### PR TITLE
Add Computation struct

### PR DESCRIPTION
**Pull request by riftEmber**

### CODE REVIEW

**Comment by riftEmber** [comment]
I followed the example of other files for the doxygen for the Computation.h file, but the one thing I wasn't sure about was what, if anything, to put in the Copyright section. What would be good to put there?

File: src/computation/Computation.cc [hunk 1]
```diff
@@ -0,0 +1,96 @@
+#include "Computation.h"
```
Comment (mstrout) [inline src/computation/Computation.cc:21]: Every .h and .cc file needs a comment header.

File: src/computation/Computation.cc [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+void Computation::printInfo() {
+    std::ostringstream stmts;
+    std::ostringstream iterSpaces;
+    std::ostringstream execSchedules;
+    std::ostringstream dataReads;
+    std::ostringstream dataWrites;
+    std::ostringstream dataSpacesOutput;
+
+    for (const auto& it : stmtsInfoMap) {
```
Comment (mstrout) [inline src/computation/Computation.cc:43]: Make sure you are only using spaces.  No tabs allowed.  I can't tell in the viewer, so please just check in your editor.

File: src/computation/Computation.h [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+ *
+ * The Computation struct is the SPF representation of a logical computation.
+ * It contains a StmtInfo struct for each statement, which in turn contains
+ * information about that statement as mathematical objects.
+ * Originally part of spf-ie.
+ *
+ * \date Started: 10/09/20
+ *
+ * \authors Anna Rift
+ */
```
Comment (mstrout) [inline src/computation/Computation.h:19]: For now it is fine to put the copyright date as University of Arizona and Boise State University 2020.

File: src/computation/Computation.h [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+struct StmtInfo {
+    //! Construct a StmtInfo given strings that will be used to construct each
+    //! set/relation.
+    StmtInfo(
+        std::string stmtSourceCode, std::string iterationSpaceStr,
+        std::string executionScheduleStr,
+        std::vector<std::pair<std::string, std::string>> dataReadsStrs,
+        std::vector<std::pair<std::string, std::string>> dataWritesStrs);
+
+    //! Source code of the statement, for debugging purposes
```
Comment (mstrout) [inline src/computation/Computation.h:123]: Nice comments!

**Review by mstrout — CHANGES_REQUESTED** [review]

File: src/computation/Computation.cc (hunk 2 above)
Comment (riftEmber) [reply src/computation/Computation.cc:43]: Yep, all soft tabs (of 4 spaces)

**Comment by riftEmber** [comment]
Suggested changes are done

**Pull request merged**